api/spod/v1alpha1/spod_types.go

```diff
@@ -33,6 +33,14 @@ type SelinuxOptions struct {
 	AllowedSystemProfiles []string `json:"allowedSystemProfiles,omitempty"`
 }
 
+// SecurityContext define the security context of a container part of SPOD daemon pod
+type SecurityContext struct {
```

Comment:
Is it a good idea to name the struct the same as the one coming from `corev1` ?

Reply:
I am open for a suggestion if you might think that this can create some confusion.

Reply:
What about making it either prefixed with `SPO` or such or naming it after the fact that it pairs the container and the contex like `SpoContSecCtx`.